dod-advana/gamechanger-web
# Task/uot 110917

## Description

-Got external API working again for searching clone
-Associated API keys to clones. They can now only search the clones they are associated to
-Made UI adjustments to admin API key table and API key request form to include clones
-Added date filter to external API

## Related Issue/Ticket

UOT-110917

## Testing instructions

-In Decoupled, go to footer and select 'Request API Key' and fill out form to request one.
-In admin panel under 'Manage API Keys', approve the request made.
-From port 8990 go to URL /api/gamechanger/external/docs/ and fill out form to make API requests
-Only one API Key should be able to be approved per user
-API keys should get back a 403 when trying to search clones they are not associated to
